wagtail/wagtailredirects/middleware.py

```diff
@@ -1,19 +1,13 @@
 from __future__ import absolute_import, unicode_literals
 
-import django
 from django import http
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
This is a Django <1.10 compatibility shim https://docs.djangoproject.com/en/1.11/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware and should probably be eliminated too now.